config/config-onprem.json

```diff
@@ -0,0 +1,6 @@
+{
+	"AB_SERVER": "{{ platform_host }}",
+	"AB_SERVER_PROTO": "https",
+	"DEBUG": false,
+	"USE_CDN_FOR_EXTENSION_DOWNLOAD": false
```

Comment:
in config-base, the value is false, so in case we use this configuration and merge it with config-base, this line is not required.